src/main/resources/stashpullrequestbuilder/stashpullrequestbuilder/StashBuildTrigger/help-ciBuildPhrases.html

```diff
@@ -0,0 +1,6 @@
+<div>
+    Build the PR if the specified phrase has been posted as a PR comment
+    in Stash, even if the PR has not changed.<br/>
```

Comment:
Actually is it a regex or literal match? Not sure myself, would have to check the code

Reply:
Actually it's a case-insensitive substring match https://github.com/jenkinsci/stash-pullrequest-builder-plugin/blob/master/src/main/java/stashpullrequestbuilder/stashpullrequestbuilder/StashRepository.java#L396

Reply:
I confirm it. But I'm not sure we should document implementation details if chances are we would change them.

Reply:
I would rather explain the current behavior.

Changing this properly should involve a data migration anyway, it can be used to convert the old value to a regex

Reply:
@proski so what about this? I'm not insisting, just making sure you didn't miss the comment above

Reply:
Done